gradle.properties

```diff
@@ -3,14 +3,14 @@ org.gradle.jvmargs=-Xmx2G
 
 # Fabric Properties
 # check these on https://fabricmc.net/develop
-minecraft_version=1.20.2
-yarn_mappings=1.20.2+build.4
-loader_version=0.14.23
+minecraft_version=1.20.4
+yarn_mappings=1.20.4+build.1
+loader_version=0.15.0
 
 #Fabric api
-fabric_version=0.90.0+1.20.2
+fabric_version=0.91.1+1.20.4
 
 # Mod Properties
-mod_version = 1.1.1+dev
+mod_version = 1.1.2+dev
```

Comment:
It is unnecessary to bump the version as 1.1.1 doesn't exist yet.

Reply:
resolved in [07191ee](https://github.com/RelativityMC/raise-sound-limit-simplified/pull/7/commits/07191eed6c24466de1475e2674cd9080a78e0670)